package.json

```
@@ -8,7 +8,7 @@
   ],
   "license": "MIT",
   "scripts": {
-    "build": "gatsby build",
+    "build": "eslint src/**/*.js && gatsby build",
```

I think we can run a script inside another script that way you will only have to modify the main script:
```suggestion
    "build": "npm run lint && gatsby build",
```